examples/client/lwm2mclient.c

```diff
@@ -95,6 +95,7 @@ lwm2m_object_t * objArray[OBJ_COUNT];
 # define BACKUP_OBJECT_COUNT 2
 lwm2m_object_t * backupObjectArray[BACKUP_OBJECT_COUNT];
 
+// Structure to hold client data
```

Comment:
We should add a comment describing what does structure `lwm2m_context_t` of type `lwm2mH` contains. If we have this already nicely described in the source code, reference would be enough